DependencyInjection/Configuration.php

```diff
@@ -76,6 +76,7 @@ public function getConfigTreeBuilder()
                     ->addDefaultsIfNotSet()
                     ->children()
                         ->booleanNode('html5_validate')->defaultValue(true)->end()
+                        ->booleanNode('admin_sort')->defaultValue(false)->end()
```

Comment:
`defaultValue(false)` can be replaced by `defaultFalse()`.

Please see: https://github.com/symfony/Config/blob/2.3/Definition/Builder/NodeDefinition.php#L241